CHANGELOG

```diff
@@ -6,6 +6,91 @@ devel
 * Foxx routes now always have a Swagger `operationId`. If the route is unnamed, a distinct operationId will be generated based on the HTTP method and URL.
 
 * coordinator code was reporting rocksdb error codes, but not the associated detail message.
+
+* Added initial support for wgs84 reference ellipsoid in GEO_DISTANCE through third
+  optional parameter to AQL function
+
+* Added support for area calculations with GEO_AREA AQL function
+
+* Added resign leadership job to supervision.
+
+* Keep followers in sync if the old leader resigned and stopped writes.
+
+* Abort a FailedLeader job when its _to_ server fails.
+
+* Removed content from Documentation/Books, but keeping the subfolders.
+  The documentation is in a separate repository (except DocuBlocks and Scripts):
+  https://github.com/arangodb/docs.git
+  
+* Updated TOKENS function to deal with primitive types and arrays.
+
+
+v3.5.0-rc.7 (2019-08-01)
```

Comment:
Are those changelogs entries also present in the 3.5 branch?

Reply:
The CHANGELOG is up to date with devel and only adds two entries.

Reply:
For the record, this is aiming for a merge into 3.6, not 3.5.